how-to/workspace-platform-starter/client/src/framework/shapes/app-shapes.ts

```diff
@@ -116,10 +116,53 @@ export interface LaunchPreference {
  * Additional options that apply to a view
  */
 export interface ViewLaunchPreference {
+	/**
+	 * If specified it indicates wish to specify specific host settings for this content.
+	 */
+	host?: HostLaunchPreference;
+}
+
+/**
+ * Additional options that apply to the host of the content
+ */
+export interface HostLaunchPreference {
 	/**
 	 * If specified it indicates you do not want to use a browser window for this view but a platform window.
 	 */
-	hostUrl?: string;
+	url?: string;
+
+	/**
+	 * If specified it indicates a preference to be used by this type of host.
+	 */
+	title?: string;
+
+	/** The Icon you would prefer the window shows. */
+	icon?: string;
+
+	/**
+	 * Should the header for the content be hidden
+	 */
+	hasHeader?: boolean;
```

Comment:
Rename to `hasHeaders` for consistency, as these are the Host preferences

Reply:
done